ubuntu16/Dockerfile

```diff
@@ -1,15 +1,21 @@
 FROM ubuntu:xenial
 
-RUN apt-get update && apt-get install -y \
+RUN dpkg --add-architecture i386 \

```

Comment:
Not sure if we want to split this into multiple RUN statements

Reply:
I don't think it matters to much.  This seems fine and it will make the container have less layers.